.github/workflows/main.yml

```
@@ -0,0 +1,77 @@
+name: Generate new docs
+
+on:
+  workflow_dispatch:
+    inputs:
+      project:
+        description: 'ember or ember-data'     
+        required: true
+      version:
+        description: 'The version to build, i.e. 3.20.0'     
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Clone the project repo
+        run: |
+          set -euo pipefail
+          if [$INPUT_PROJECT -eq 'ember']
+          then
+            git clone https://github.com/emberjs/ember.js
+            cd ember.js
+            git checkout "v$INPUT_VERSION"
+          elif [$INPUT_PROJECT -eq 'ember-data']
+          then
+            git clone https://github.com/emberjs/data
+            cd data
+            git checkout "v$INPUT_VERSION"
+          else
+            echo "Please specify either ember or ember-data for the project."
+          fi
```

I think this can be replaced by the actions using the checkout actions again, something like

```suggestion
      - name: Clone ember.js
        uses: actions/checkout@v2
        if: ${{ github.event.inputs.project == "ember" }}
        with:
          repository: emberjs/ember.js
          path: ember.js
          ref: v${{ github.event.inputs.version }}

      - name: Clone ember-data
        uses: actions/checkout@v2
        if: ${{ github.event.inputs.project == "ember-data" }}
        with:
          repository: emberjs/data
          path: data
          ref: v${{ github.event.inputs.version }}
```